README.md

```diff
@@ -69,22 +69,28 @@ format={"additional_field": "value"}
  Please note, that you have to configure those parameters by this exact order.
  i.e. you cannot set Debug to true, without configuring all of the previous parameters as well.
 
+#### Serverless platforms
+In case you are using a serverless platform, you'll need to import and add the LogzioFlusher annotation before your sender function. To use it in the code example below, uncomment the import statement and the `import` statement and the `@LogzioFlusher(logger)` line.
```

Comment:
```suggestion
In case you are using a serverless function, you'll need to import and add the LogzioFlusher annotation before your sender function. To use it in the code example below, uncomment the import statement and the `import` statement and the `@LogzioFlusher(logger)` line.
```

Reply:
NO